src/analysis/quaternion.lean

```diff
@@ -154,4 +154,24 @@ begin
   exact (complete_space_congr this).1 (by apply_instance)
 end
 
+@[simp, norm_cast] lemma has_sum_coe {f : α → ℝ} {r : ℝ} :
+  has_sum (λ a, (f a : ℍ[ℝ])) (↑r : ℍ[ℝ]) ↔ has_sum f r :=
+⟨λ h, by simpa only using
+  h.map (show ℍ[ℝ] →ₗ[ℝ] ℝ, from quaternion_algebra.re_lm _ _) continuous_re,
+  λ h, by simpa only using h.map (algebra_map ℝ ℍ[ℝ]) (continuous_algebra_map _ _)⟩
+
+@[simp, norm_cast]
+lemma summable_coe {f : α → ℝ} : summable (λ a, (f a : ℍ[ℝ])) ↔ summable f :=
+by simpa only using summable.map_iff_of_left_inverse (algebra_map ℝ ℍ[ℝ])
+  (show ℍ[ℝ] →ₗ[ℝ] ℝ, from quaternion_algebra.re_lm _ _)
+  (continuous_algebra_map _ _) continuous_re coe_re
+
+@[norm_cast] lemma tsum_coe (f : α → ℝ) : ∑' a, (f a : ℍ[ℝ]) = ↑(∑' a, f a) :=
```

Comment:
This lemma is still the wrong way around for `norm_cast`, isn't it?

Reply:
`norm_cast` doesn't care, it works out which is the "right way" around by introspection I believe.

Reply:
Oh, okay